CTP/shell/src/com/navercorp/cubridqa/shell/main/GeneralLocalTest.java

```diff
@@ -15,15 +15,15 @@
 
 	public GeneralLocalTest(Context context) {
 		this.context = context;
-		this.context.setLogDir(context.getProperty("main.testing.category", "general"));
-		String feedbackType = context.getProperty("main.feedback.type", "").trim();
+		this.context.setLogDir(context.getTestCategory());
```

Comment:
as is :
this.context.setLogDir(context.getTestCategory());

to be:
if(isEmpty(context.getTestCategory()) {
        this.context.setLogDir("general");
} else {
        this.context.setLogDir(context.getTestCategory());
}